SuiteCRM/SuiteCRM
# Import By Email Address

## Description




Adds a new option to the Contacts module Import to allow records to be matched based on the row's email address when updating, rather than just the ID. Only on the Contacts module at this time, but can be easily extended to other modules that use Email Addresses. The matching will take the first Contact it finds with that email address.

## Motivation and Context

Provides a new method of importing Contacts and updating them based on their email address, if the CRM ID is not available.

## How To Test This

1. Prepare a CSV with an email address included in the data.
2. Go to the Contacts Import view.
3. Select the new third option to import and update by email address.
4. Go through the regular import steps.
5. Once imported, check results to see newly created contacts or updates to existing ones are correct.

## Types of changes


### Final checklist
